contracts/characters.sol

```diff
@@ -95,6 +97,8 @@ contract Characters is Initializable, ERC721Upgradeable, AccessControlUpgradeabl
 
     Promos public promos;
 
+    uint characterLimit;
```

Comment:
`uint` is shorthand for `uint256`, and I realize the original code in `cryptoblades.sol` itself used `uint` for this, but I think we should prefer explicitly stating it.

```suggestion
    uint256 characterLimit;
```